File: leetcode-submissions/3014_Minimum_Number_of_Pushes_to_Type_Word_I.ts
function minimumPushes(word: string): number {
    let ans:number=0;
    let cnt:number=1;
    const size:number=word.length;
    for(let i=0;i<size;i+=8){
        ans+=cnt*Math.min(8,size-i);
        cnt++;
    }
    return ans;
};

File: leetcode-submissions/3014_Minimum_Number_of_Pushes_to_Type_Word_I.cpp
class Solution {
public:
    int minimumPushes(string word) {
        int cnt=1;
        int size=word.size();
        int ans=0;

        for(int i=0;i<size;i+=8){
            ans+=cnt*min(8,size-i);
            cnt++;
        }
        return ans;
    }
};